Add changelog (#423)
CloudCannon/platform-documentation: changelogs/2025-01-17_better-accessibility-in-the-content-editor.mdx

---
_schema: default
title: Better accessibility in the Content Editor
---
This release added more keyboard controls for the Content Editor and better handling in the terminal output.

It also addressed several issues, including those affecting avatars and Organization invites.

## Features & Improvements

* Better accessibility for the WYSIWYG toolbar in the Content Editor.
* Better handling of carriage returns in the terminal output.

## Fixes

* Fixed an issue causing unexpected avatar behavior when keeping track of which users are looking at a Site or file.

> * Fixed an issue preventing users with a capital letter in their email address from receiving invitations to Organizations.

Reviewer: ```suggestion
* Fixed an issue preventing invitations to Organizations being sent if a capital letter was used in the email address.
```
